src/commands/dev/index.js

```diff
@@ -223,7 +213,9 @@ class DevCommand extends Command {
     const liveTunnelUrl = await handleLiveTunnel({ flags, site, api, settings, log })
     url = liveTunnelUrl || url
 
-    await openBrowser({ devConfig, url, warn })
+    if (devConfig.autoLaunch !== false) {
+      await openBrowser({ url, log, silentBrowserNoneError: true })
```

Comment:
In all other cases we print an error when `BROWSER=none` is set as an env variable. When running `netlify dev` is it quite useful to run `BROWSER=none netlify dev` and should not be considered as an error.